tools/check_lint_rules_identity/lib/src/utils/version_utils.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:pub_semver/pub_semver.dart';
+
+// Version.parse cannot be used if minor does not exist, so I created my own parse method.
```

Comment:
![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
Could this be a mistake for patch instead of minor?
```suggestion
// Version.parse cannot be used if patch does not exist, so I created my own parse method.
```

Reply:
Oh...sorry, It's my mistake.
I fixed. 403f5426b547b59aa839074f83e13d2f8617785a